codalab/lib/codalab_manager.py

```diff
@@ -368,8 +368,8 @@ def emailer(self):
         if 'email' in self.config:
             return SMTPEmailer(
                 host=self.config['email']['host'],
-                user=self.config['email']['user'],
-                password=self.config['email']['password'],
+                user=self.config['email'].get('user','noreply@codalab.org'),#['user'],
+                password=self.config['email'].get('password',None),#['password'],
```

Comment:
Could you remove everything that's commented out? And also add a comment describing why you made those changes (the description you wrote in the conversation is fine, i.e. "Some SMTP servers don't require u/p authentication; these flags allow such a server to be used with CodaLab.")